src/core/Notification/Notification.tsx

```diff
@@ -20,7 +20,8 @@ export const baseClassName = 'fi-notification';
 export const notificationClassNames = {
   styleWrapper: `${baseClassName}_style-wrapper`,
   content: `${baseClassName}_content`,
-  label: `${baseClassName}_label`,
+  innerContent: `${baseClassName}_innerContent`,
```

Comment:
Would `mainContent` describe the element better? Might work with the current name just fine, but just thinking out loud here.

Reply:
innerContent is child of content, thats why I selected that name...

Reply:
The API description is `Main content of the Notification` and type is ReactNode. In addition, there is a parent node with className `textContentWrapper` which seems to be incorrect as ReactNode can contain almost anything. How about using just `contentWrapper`, `content` and, if needed, `mainContent` as described in the API. Another option would be to rename the contentWrapper to something like liveRegionWrapper etc.

Reply:
I changed it to contentWrapper, that is better name.